x-pack/plugins/security_solution/public/common/components/inspect/index.tsx

```diff
@@ -71,6 +72,7 @@ const InspectButtonComponent: React.FC<InspectButtonProps> = ({
   isInspected,
   loading,
   inspectIndex = 0,
+  multiple = false,
```

Comment:
I added a new prop _multiple_ to show all the requests/responses in the inspect modal. when `multiple = true` it ignores the _inspectIndex_ and passes all requests/responses to the inspect modal.